src/datapump/jobs/version_update.py

```diff
@@ -30,6 +30,7 @@ class RasterVersionUpdateJob(Job):
     dataset: str
     version: str
     content_date_range: ContentDateRange
+    content_date_description: str
```

Comment:
Looks good, is this always required? If not, we could default to null or empty string